docs/user/security/audit-logging.asciidoc

```diff
@@ -123,18 +123,18 @@ Refer to the corresponding {es} logs for potential write errors.
 | `unknown` | User is updating a saved object.
 | `failure` | User is not authorized to update a saved object.
 
-.2+| `saved_object_add_to_spaces`
-| `unknown` | User is adding a saved object to other spaces.
-| `failure` | User is not authorized to add a saved object to other spaces.
-
-.2+| `saved_object_delete_from_spaces`
-| `unknown` | User is removing a saved object from other spaces.
-| `failure` | User is not authorized to remove a saved object from other spaces.
+.2+| `saved_object_update_objects_spaces`
```

Comment:
@gchaps A couple of updates were made here to reflect the current authorization audit actions.